### PR TITLE
feat: VOL-5664 selfserve accounts require t&c signup on first login

### PR DESCRIPTION
**Pull request by ilindsay**
## Description

Related issue: [VOL-5664](https://dvsa.atlassian.net/browse/VOL-5664)

### CODE REVIEW

File: src/Command/User/AgreeTerms.php [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+use Dvsa\Olcs\Transfer\FieldType\Traits\TranslateToWelshOptional;
+use Dvsa\Olcs\Transfer\Util\Annotation as Transfer;
+use Dvsa\Olcs\Transfer\Command\AbstractCommand;
+
+/**
+ * @Transfer\RouteName("backend/user/selfserve/agree-terms")
+ * @Transfer\Method("PUT")
+ */
+final class AgreeTerms extends AbstractCommand
+{
```
Comment (fibble) [inline src/Command/User/AgreeTerms.php:17]: is this not missing a `use Identity;` ?

Comment (fibble) [inline src/Command/User/AgreeTerms.php:17]:
```suggestion
{
    use Identity;
```

File: src/Command/User/AgreeTerms.php [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Dvsa\Olcs\Transfer\Command\User;
+
+use Dvsa\Olcs\Transfer\FieldType\Traits\Identity;
+use Dvsa\Olcs\Transfer\FieldType\Traits\TranslateToWelshOptional;
```
Comment (fibble) [inline src/Command/User/AgreeTerms.php:8]:
```suggestion
```

File: src/Command/User/AgreeTerms.php (hunk 1 above)
Comment (ilindsay) [reply src/Command/User/AgreeTerms.php:15]: Yes that's correct. I'd originally put it in, and then realised it was pointless as needed to check who the logged in user was on the API side in any case.

**Review by fibble — APPROVED** [review]

**Review by sdh100shaun — APPROVED** [review]

**Pull request merged**